ldes-server-port-ingest-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingest/rest/validators/memberingestvalidator/MemberIngestValidator.java

```diff
@@ -45,16 +46,38 @@ private void validateModel(Model model, EventStream eventStream) {
         if (memberSubjects.size() > 1 && !eventStream.isVersionCreationEnabled()) {
             throw new IngestValidationException("Only 1 member is allowed per ingest");
         }
-        int timestampPathCount = model.listStatements(null, ResourceFactory.createProperty(eventStream.getTimestampPath()), (RDFNode) null).filterKeep(statement -> memberSubjects.contains(statement.getSubject())).toList().size();
-        int versionOfPathCount = model.listStatements(null, ResourceFactory.createProperty(eventStream.getVersionOfPath()), (RDFNode) null).filterKeep(statement -> memberSubjects.contains(statement.getSubject())).toList().size();
-        boolean conformsTimePath = eventStream.isVersionCreationEnabled() ? timestampPathCount == 0 : timestampPathCount == 1;
-        boolean conformsVersionPath = eventStream.isVersionCreationEnabled() ? versionOfPathCount == 0 : versionOfPathCount == 1;
+
+        boolean conformsTimePath = validateTimestampPath(memberSubjects, model, eventStream);
+        boolean conformsVersionPath = validateVersionOfPath(memberSubjects, model, eventStream);
         if (!(conformsTimePath && conformsVersionPath)) {
             String message = getValidationMessage(eventStream, conformsTimePath, conformsVersionPath);
             throw new IngestValidationException(message);
         }
     }
 
+    private boolean validateTimestampPath(List<Resource> memberSubjects, Model model, EventStream eventStream) {
+        List<Statement> timestampStatements = model.listStatements(null, ResourceFactory.createProperty(eventStream.getTimestampPath()), (RDFNode) null).filterKeep(statement -> memberSubjects.contains(statement.getSubject())).toList();
+
```

Comment:
I am going to leave it as a suggestion and not an action point, but a method that extract those statements with arguments that includes the timestampPath, so that that method also could be used for lines 71-72

Reply:
But I was wondering if you could "custom" format these two lines (and lines 71-72) so that they can fit on a single line